data_structures/test_queue_with_stacks.py

```diff
@@ -0,0 +1,64 @@
+from queue_with_stacks import Queue_with_stacks as ST
+import pytest
+
+
+def test_init(empty_stack):
+    """initiate with no value"""
+    assert empty_stack.stack_back.top == None
+
+
+def test_init_two():
+    """initiate with value"""
+    teststack2 = ST([1, 2, 3, 4, 5])
+    assert teststack2.stack_back.top.val == 1   
+
+
+def test_init_three(iter_stack):
+    """initiate with value stack front"""
+    assert iter_stack.stack_front.top is None
+    pass
+
+
+def test_enqueue(empty_stack):
+    """enqueue with empty stack"""
+    empty_stack.enqueue(1)
+    assert empty_stack.stack_back.top.val == 1
+
+
+def test_enqueue_two(empty_stack):
+    """enqueue two with empty stack"""
+    empty_stack.enqueue(1)
+    empty_stack.enqueue(2)
+    assert empty_stack.stack_back.top.val == 2   
+
+
+def test_enqueue(empty_stack):
+    """enqueue multiple with empty stack"""
+    empty_stack.enqueue(1)
+    empty_stack.enqueue(2)
+    empty_stack.enqueue(3)
+    assert empty_stack.stack_back.top.val == 3 
+
+
+def test_dequeue(empty_stack):
+    """test dequeue returns first in"""
+    empty_stack.enqueue(1)
+    empty_stack.enqueue(2)
+    empty_stack.enqueue(3)
+    assert empty_stack.dequeue() == 1   
+
+
+def test_dequeue(empty_stack):
```

Comment:
You need to have different names for all of your test functions or they will not be run.